avaframe/com1DFA/com1DFACfg.ini

```diff
@@ -105,11 +105,13 @@ entTh =
 dt = 0.1
 # End time [s]
 tEnd = 400
+# only one of the two following options can be activated
 # to use a variable time step (defined by the CFL number)
 cflTimeStepping = False
 # to use a variable time step (time step depends on kernel radius)
 sphKernelRadiusTimeStepping = False
```

Comment:
This looks like an easy way to accidentally mix things up. I suggest switching to one variable with options, i.e:

timeStepping = cflTime
or 
timeStepping = sphKernelRadius

Reply:
I think in this case it needs adding an option for using just dt as a fixed time step now which is our default currently